assets/js/mediavalidation.js

```diff
@@ -0,0 +1,20 @@
+/*const fileHeaders = {
```

Comment:
This file serves no present purpose. I would instead suggest using this file to implement the image load callback which is currently in upload.js.